docs/source/howto/installation.rst

```diff
@@ -223,24 +223,327 @@ the configuration directory ``~/project_a/.aiida`` will be used.
+
+        In order to exclude the repository folder from indexing, add its path to the ``PRUNEPATH`` variable in the ``/etc/updatedb.conf`` configuration file (use ``sudo``).
+
+
+  4. The verdi deamon can manage an arbitrary number of parallel workers; by default only one is activated. If ``verdi daemon status`` shows the daemon worker(s) constantly at high CPU usage, use ``verdi daemon incr X`` to add ``X`` daemon workers.
+     It is recommended that the number of workers does not exceed the number of CPU cores (or, if possible, that you use one or two cores less than your machine has, to avoid to degrade the PostgreSQL database performance).
 
 .. _how-to:installation:update:
 
 Updating your installation
 ==========================
 
-`#4003`_
+1. Activate the Python environment where AiiDA is installed.
+2. Finish all running calculations.
+   All finished calculations will be automatically migrated, but it is not possible to resume unfinished calculations.
+3. Stop the daemon using ``verdi daemon stop``.
+4. :ref:`Create a backup of your database and repository<how-to:installation:backup>`.
+
+   .. warning::
+
+      Once you have migrated your database, you can no longer go back to an older version of ``aiida-core`` (unless you restore your database and repository from a backup).
+
+5. Update your ``aiida-core`` installation.
+
+    * If you have installed AiiDA through ``conda`` simply run: ``conda update aiida-core``.
+    * If you have installed AiiDA through ``pip`` simply run: ``pip install --upgrade aiida-core``.
+    * If you have installed from the git repository using ``pip install -e .``, first delete all the ``.pyc`` files (``find . -name "*.pyc" -delete``) before updating your branch with ``git pull``.
+
+6. Migrate your database with ``verdi -p <profile_name> database migrate``.
+   Depending on the size of your database and the number of migrations to perform, data migration can take time, so please be patient.
+
+After the database migration finishes, you will be able to continue working with your existing data.
+
+.. note::
+    If the update involved a change in the major version number of ``aiida-core``, expect :ref:`backwards incompatible changes<updating_backward_incompatible_changes>` and check whether you also need to update installed plugin packages.
+
+Updating from 0.x.* to 1.*
+--------------------------
+- `Additional instructions on how to migrate from 0.12.x versions <https://aiida.readthedocs.io/projects/aiida-core/en/v1.2.1/install/updating_installation.html#updating-from-0-12-to-1>`_.
+- `Additional instructions on how to migrate from versions 0.4 -- 0.11 <https://aiida.readthedocs.io/projects/aiida-core/en/v1.2.1/install/updating_installation.html#older-versions>`_.
+- For a list of breaking changes between the 0.x and the 1.x series of AiiDA, check `this page <https://aiida.readthedocs.io/projects/aiida-core/en/v1.2.1/install/updating_installation.html#breaking-changes-from-0-12-to-1>`_.
```

Comment:
The other approach to this is to keep move these sections to e.g.  `docs/source/howto/_version_migrations.rst` and link to that.
Then at the top of that document put `:orphan:`; this tells sphinx that the document is not required to show up in any ToC. This is what I have currently done in #4060 (see https://7-77234579-gh.circle-artifacts.com/0/html/intro/installation.html#updating-aiida-core), but I will remove that section and link here